aspnet/fundamentals/logging/sample/src/TodoApi/Core/LoggingEvents.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace TodoApi.Core
```

Comment:
FYI, for samples I've been removing the namespace, then I don't need to use snippet tags.